.github/workflows/update-screenshots.yml

```
@@ -49,17 +49,25 @@ jobs:
           DEBIAN_FRONTEND=noninteractive sudo apt-get install --no-install-recommends -yq \
             libasound2-dev libudev-dev libxkbcommon-x11-0;
 
-      - name: install xvfb, llvmpipe and lavapipe
+      - name: Install xvfb, llvmpipe and lavapipe
         run: |
           sudo apt-get update -y -qq
           sudo add-apt-repository ppa:kisak/turtle -y
           sudo apt-get update
           sudo apt install -y xvfb libegl1-mesa libgl1-mesa-dri libxcb-xfixes0-dev mesa-vulkan-drivers
 
+      - name: Install oxipng
+        run: |
+          cargo install oxipng
```

Could you please pin `oxipng` to be version 9? This prevents breaking changes from an eventual version 10.0.0 from hurting us later. I believe it would look like this:

```suggestion
        run: |
          cargo install oxipng --version ^9.1.1
```

I believe that will allow versions >= 9.1.1 and < 10.0.0, but please test it first!